src/styles/_view_lineup.scss

```diff
@@ -302,12 +302,20 @@ $lu_assets: '~lineupjs/src/assets';
         bottom: 0;
         right: 0;
         left: 0;
-        overflow-y: auto; // make tab pane scrollable for long content
-      }
+        flex-direction: column;
 
-      // default = LineUp Ranking Configuration tab
-      .tab-pane.default {
-        overflow-y: hidden; // disable scrolling of tab pane, as only the column filters are scrollable
+        // Add scrollbar to SidePanel higher in the node tree
+        .lu-side-panel-main {
+          overflow-y: auto;
+
+          .lu-side-panel-ranking-main {
+            overflow: visible;
+          }
+        }
+
+        &.active {
+          display: flex;
+        }
```

Comment:
I would move this directly after line 305, to make it clearer that these things belong together.